vendor/bat-native-ledger/src/bat_contribution.cc

```diff
@@ -1210,6 +1220,12 @@ void BatContribution::ProofBatch(
     proofs.push_back(annon_proof);
   }
 
+  return proofs;
+}
+
+void BatContribution::ProofBatchCallback(
+    const braveledger_bat_helper::BathProofs& batch_proof,
```

Comment:
nit: `batch_proof`s

Reply:
I know it was like this before, but with `s` sounds better

Reply:
fixed with https://github.com/brave/brave-core/pull/1800/commits/ec256f8a5190fec5870301860736a9a9e7cbc7ba, also fix BatchProofs typo